### DISCUSSION TITLE
How can I pay for my previous order?

### DISCUSSION THREAD

**Discussion by Blushyes** [community/community#147962] [Actions]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I previously switched my GitHub plan to a paid plan, but I forgot it was a monthly payment, so I didn't have enough money on my payment card. After receiving three messages, I finally realized I hadn't paid, which resulted in my GitHub Actions being disabled. I have now added sufficient funds to my card, but how do I pay this outstanding bill so I can use GitHub Actions normally again?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ebndev** [comment]
Hi @Blushyes,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
